host-controller/src/main/java/org/jboss/as/host/controller/mgmt/HostInfo.java

```diff
@@ -130,6 +135,7 @@ public static HostInfo fromModelNode(final ModelNode hostInfo, DomainHostExclude
     private final Set<ServerConfigInfo> serverConfigInfos;
     private final Set<String> domainIgnoredExtensions;
     private final boolean hostDeclaredIgnoreUnaffected;
+    private final FeatureStream stream;
```

Comment:
The HostInfo must override the `getFeatureStream()` method and return this steam value instead of the one supplied by default by its parent `org.jboss.as.controller.Feature` class.

Reply:
Indeed.  Good catch.

Reply:
Fixed.

Reply:
Thanks!